### PR TITLE
Recursive backup test

### PR DESCRIPTION
**Pull request by aayushKumarJarvis**

### CODE REVIEW

**Comment by ashwanthkumar** [comment]
@aayushKumarJarvis There seems to be some merge conflicts. Can you please resolve them and update the PR?

**Comment by aayushKumarJarvis** [comment]
Updated the PR. Can you try merging again ?

File: ops/backup.go [hunk 1]
```diff
@@ -50,8 +50,9 @@ func walkSourceDir(source, destination string) {
 
 			dbBackupLoc := filepath.Join(destination, dbRelative)
 			log.Printf("Backup at %s, would be stored to %s\n", dbLoc, dbBackupLoc)
-
+			log.Printf("Backup is created for %s rocks store", dbRelative)
```
Comment (ashwanthkumar) [inline ops/backup.go:53]: @aayushKumarJarvis We've 2 instances of `log.Printf`

**Comment by ashwanthkumar** [comment]
@aayushKumarJarvis the changes are too small, I'm merging this and fixing it later.

**Pull request merged**